docs-kits/kits/Digital Twin Kit/page_software-operation-view.md

```diff
@@ -23,16 +23,21 @@ CX-0002.
+| Service Name          | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          | Reference Implementation                                                                                                                                                                                        | [Standardized in](https://catena-x.net/de/standard-library) |
+|-----------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------|
+| Discovery Finder      | A microservice resolving a type of specific asset identifiers, e.g. `manufacturerPartId` or `intrinsicId` against a set of BPN-, and EDC-Discovery Servers hosted at Catena-X operating companies. These identifiers are often shared across use-cases (see [Industry Core](../Industry%20Core%20Kit/Software%20Development%20View)) but are not specified by the DT Kit and standards.                                                                                                                                                                                                                                                              | [Tractus-X Discovery Finder](https://github.com/eclipse-tractusx/sldt-discovery-finder)                                                                                                                         | CX - 0053                                                   |
```

Comment:
Typo: instanceId instead of intrinsicId?

Reply:
nope, this is a "new" specificAssetId that was introduced by the industry core in R24-05